src/utilities/viewer.cpp

```diff
@@ -217,15 +215,29 @@ class TestViewerWindow final : private D3D11Resources {
       .lpszClassName = CLASS_NAME,
     };
     RegisterClass(&wc);
+
+    POINT TOP_LEFT = POINT(mSettings.mWindowX, mSettings.mWindowY);

```

Comment:
```C++
const POINT topLeft { mSettings.mWindowX, mSettings.mWindowY };
```

- Local variables should usually be `lowerCamelCase` ; `UPPERCASE` should be just for macros, except where windows has required otherwise
- use `const` when possible (so also for the monitor variables below)

There's a few cases where I'm using them for constants/enum values too, but that's really just something I haven't got around to cleaning up yet.